app/views/devise/registrations/new.html.haml

```diff
@@ -1,4 +1,4 @@
-%h2 Sign up
+%h2.title Sign up
```

Comment:
You also has a mixture between herb and haml.
Maybe not changing it in this PR, but remember, keep consistency is important

Reply:
Sure, actually I'll open a new PR to replace haml by erb